workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/views/static_report.py

```diff
@@ -439,7 +430,37 @@ def main(args):
     }
 
     for filename, html in pages.items():
-        upload(html, os.path.join(args.output, filename))
+        upload(html, os.path.join(output, filename))
+
+
+def register_parser(subparsers):
+    parser = subparsers.add_parser("report", help="Generate a static html report.")
+    parser.add_argument("input", help="Directory containing multiple run diagnostics.")
+    parser.add_argument("output", help="Location to save report html files.")
+    parser.set_defaults(func=main)
+
+    parser = subparsers.add_parser(
```

Comment:
use a different local variable name for this parser?

Reply:
I split this into helpers to clarify the distinction.